README.rst

```diff
@@ -75,6 +75,36 @@ Using `yaourt`::
 
     $> yaourt -S python2-pythran-git
 
+Windows
+=======
+
+Using WinPython (http://winpython.github.io), installed in ``E:\WinPython-32bit-2.7.9.5``
+
+1. Install the binary version of CMake (http://www.cmake.org/download), and
+   make sure to check to check the box to add cmake to the PATH.
+
+2. Install Boost (http://www.boost.org/users/download/): Once the archive is
+   extracted, use the *WinPython Command Prompt* and run the following::
+
+        cd boost_1_58
+        .\bootstrap.bat
+        .\b2 toolset=gcc link=shared --with-python
+        xcopy /E boost E:\WinPython-32bit-2.7.9.5\python-2.7.9\include\boost
+        copy stage\lib E:\WinPython-32bit-2.7.9.5\python-2.7.9\libs
+        copy stage\lib E:\WinPython-32bit-2.7.9.5\python-2.7.9\DLLs
```

Comment:
For now, I don't really use BOOST_ROOT so yes, it is still needed.
I though about your idea to remplace BOOST_ROOT at install time but I think it is tricky for users as if they just use : python setup.py install, it will not work.